vignettes/D_Crossing.Rmd

```diff
@@ -7,9 +7,9 @@ vignette: >
   %\VignetteEngine{knitr::rmarkdown}
```

Comment:
Elsewhere, you changed the name of the vignettes to just the topic without "SIMplyBee"

Reply:
Thanks for spotting. I was removing that string because I noticed a warning message from R / knitr …